docs/_posts/2019-05-08-530-release.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blogpost
+title:  "5.3.0 Release Announcement"
+section_id: blog
+date:   2019-05-08 12:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hello Open GEE Community!
+ 
+We are happy to announce the official release of Open GEE 5.3.0!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+ 
+**Performance and memory enhancements include:**
+
+**- The ability to control the XML cache size.** The size can be modified by configuring the _INIT_HEAP_SIZE_, _MAX_HEAP_SIZE_, and _BLOCK_SIZE_ settings in _/etc/opt/google/XMLparams_. This file must be created manually. View the [Configure Fusion performance document](http://www.opengee.org/geedocs/5.3.0/answer/176738.html) for more details.
+
+**- The capability of purging periodically the XML cache.** This can be done by configuring the PURGE and PURGE_LEVEL settings in /etc/opt/google/XMLparams. This file must be created manually. The details are explained in the [Configure Fusion performance document](http://www.opengee.org/geedocs/5.3.0/answer/176738.html).
```

Comment:
Formatting of PURGE and PURGE_LEVEL and /etc/opt/google/XMLparams doesn't match first bullet

Reply:
will use the same formatting